src/com/robibp/trening/UnosKlinca.java

```diff
@@ -0,0 +1,38 @@
+package com.robibp.trening;
+
+import java.util.Scanner;
+
+class UnosKlinca {
+
+    private boolean samoSlova(String ime) {
+        char[] slova = ime.toCharArray();
+
+        for (char slovo : slova) {
+            if (!Character.isLetter(slovo)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+
+    String validacijaImena() {
```

Comment:
Stavi da je `public`.

Takodjer, naziv `validacijaImena()` nije dobar. Taj bi naziv bio dobar kada bi ta metoda primila neko ime kao parametar i onda ga validirala. Uz to kad kazes validacija onda ocekujes da li je validno ili nije (znaci `boolean`), a ne da vrati `String`.

Nazovi metodu `unos()` za sada.

Reply:
I zasto vraca `String`??? Pa treba napravit klinca (imas POJO Klinac)